src/widgets/_stackable_boxes.scss

```
@@ -0,0 +1,9 @@
+deck > shadow {
```

I'd also just comment this for future spelunkers :)

```suggestion
// Hdy.Deck and Hdy.Leaflet
deck > shadow {
```